src/components/editor/AuthorListController.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController'
+
+export default class AuthorListController extends FalcorController {
+  static getFalcorPath(params) {
+    return [];
+  }
+
+  render() {
+    return <div><p>Hi</p></div>;
+  }
+}
```

Comment:
I guess it's just me that was trying to delete all trailing newlines at the end of the file, but I guess I should leave one there then?